src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/search/TwitterSearchClientSpec.scala

```diff
@@ -14,11 +14,22 @@ class TwitterSearchClientSpec extends ClientSpec {
       val result: RatedData[StatusSearch] = when(searchTweet("#scala")).expectRequest { request =>
         request.method === HttpMethods.GET
         request.uri.endpoint === "https://api.twitter.com/1.1/search/tweets.json"
-        request.uri.queryString() === Some("count=15&include_entities=true&q=%23scala&result_type=mixed")
+        request.uri.rawQueryString === Some("count=15&include_entities=true&q=%23scala&result_type=mixed")
```

Comment:
Any particular reason why you changed this? I do not mind changing it, as long as it is consistent across all the other tests

Reply:
@DanielaSfregola I changed this because it is confusing, you want actually check what raw query is, while `queryString()` converts the output from utf-8, for instance if you had `%23scala ` would convert it to `#scala`. Not sure is it wrong or right, but it is confusing anyway.

Reply:
`%23scala` is already an UTF-8 string. An example of a string non-utf-8 is a string containing an emoji.

Anyway, you are right: `rawQueryString` is less confusing: can you please change it everywhere? It should just be the case of a find/replace all.